src/handlebars/about.handlebars

```diff
@@ -7,53 +7,81 @@
 
     <div id="support-container" class="blue-link-section">
 
-      <!-- TODO this might be renamed to GetOpenJDK to avoid confusion -->
       <div class="margin-bottom">
-        <h2 class="bold">About AdoptOpenJDK - GetOpenJDK</h2>
+        <h2 class="bold">About AdoptOpenJDK.net</h2>
         <p>
-          This initiative came about following a discussion about the general lack of an open and reproducible build & test system
-          for the OpenJDK source across multiple platforms.
+          AdoptOpenJDK.net started in 2017 following years of discussions about
+          the general lack of an open and reproducible build & test system for
+          <a target="_blank" href="https://openjdk.java.net">OpenJDK</a> and
+          <a target="_blank" href="https://www.eclipse.org/openj9/">Eclipse OpenJ9</a>
+          source across multiple platforms.
```

Comment:
Let's not keep calling out OpenJDK and Eclipse OpenJ9 as a definitive list, or we soon get into JavaFX, Icedtea-web, Corretto, SAPMachine, etc. lists.
Propose just using "... build and test system for OpenJDK across multiple platforms and versions."

Reply:
Fixed